recipes/cp2k/mc/repo/packages/intel-oneapi-mkl/README.md

```diff
@@ -0,0 +1,3 @@
+# `intel-oneapi-mkl`
```

Comment:
Can you move the readme up to the `recipes/cp2k/mc/repo` path, and have all of the packages listed there, with the reason for their inclusion and notes on when it might be possible to deprecate them?
The suggestion is so that they are documented in one place.